src/gui/render/graphics_shaders/quad_renderer_multisample.cpp

```diff
@@ -0,0 +1,51 @@
+#include "quad_renderer_multisample.hpp"
+
+#include "../../../util/files.hpp"
+#include "../../meshloader.hpp"
+#include "../../shader.hpp"
+
+#include <GL/glew.h>
+#include <GLFW/glfw3.h>
+#include <glm/glm.hpp>
+
+#include <memory>
+
+namespace gui {
+
+namespace render {
+
+QuadRendererMultisample::QuadRendererMultisample(ShaderHandler shader_handler) :
+    QuadRenderer(shader_handler) {
+    // program
+    program_id_ = shader_handler.load_program(
+        files::get_resources_path() / "shaders" / "Passthrough.vert",
+        files::get_resources_path() / "shaders" / "SimpleTextureMS.frag"
+    );
+    width_id_ = glGetUniformLocation(program_id_, "width");
+    height_id_ = glGetUniformLocation(program_id_, "height");
+    tex_samples_id_ = glGetUniformLocation(program_id_, "tex_samples");
+
+}
```

Comment:
put in header

Reply:
I don't want to because this file will be recompiled a lot. I should move the program file path to be passed, in addition every time I want to change one of these strings I will need to recompile n extra files.

Reply:
alright